core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -352,13 +353,15 @@ public Response getBucket(@HeaderParam("Authorization") String authorization,
    */
   @POST
   @Path(BUCKET_PARAM)
-  public Response postBucket(@PathParam("bucket") final String bucket,
+  public Response postBucket(@HeaderParam("Authorization") String authorization,
```

Comment:
In my opinion, authorization is defined as final String is better.

Reply:
All finished.